dist/data-viz-colors/data-viz-colors.css

```diff
@@ -0,0 +1,85 @@
+.data-viz-color-primary {
+  fill: var(--color-data-viz-primary, var(--color-blue-6));
+}
+.data-viz-color-primary::before {
+  background-color: var(--color-data-viz-primary, var(--color-blue-6));
```

Comment:
We'll need to add these new data-viz token aliases to evo-light.css and evo-dark.css. If there are a LOT of them we might also consider creating separate/optional data-viz token files.

Reply:
I added the --color-data-viz-* values just so they could be overwritten if wanted, they use the fallback with the standard --color values

Reply:
First value should be the component level token (i.e. a component level override), second fallback value should be our system token (note: system tokens don't yet exist in skin for data viz, but we should add them)